README.md

```diff
@@ -1,29 +1,54 @@
+[![OpenFisca logo](https://www.openfisca.fr/hotlinks/logo-openfisca.svg)](https://www.openfisca.fr/)
+> Download an [offline version](https://www.gitbook.com/book/openfisca/documentation) of the documentation (direct link to [PDF](https://www.gitbook.com/download/pdf/book/openfisca/documentation), [ePub](https://www.gitbook.com/download/epub/book/openfisca/documentation) and [Mobi](https://www.gitbook.com/download/mobi/book/openfisca/documentation)).
+
 # About the project
 
-> You can download an [offline version](https://www.gitbook.com/book/openfisca/documentation) of the documentation (direct link to [PDF](https://www.gitbook.com/download/pdf/book/openfisca/documentation), [ePub](https://www.gitbook.com/download/epub/book/openfisca/documentation) and [Mobi](https://www.gitbook.com/download/mobi/book/openfisca/documentation)).
+[OpenFisca](https://www.openfisca.fr/) is a versatile microsimulation software.
+OpenFisca allows users to :
+- Calculate many variables of the tax and benefit system of a country (social benefits and taxes paid by households) given input variables.
+- Simulate the budgetary consequences of a reform and its distributional impact when plugged on a survey.
+Its engine is independent of the country, it is therefore possible to simulate any country.
+For now the main supported country is France.
 
-[![OpenFisca logo](https://www.openfisca.fr/hotlinks/logo-openfisca.svg)](https://www.openfisca.fr/)
+[OpenFisca](https://www.openfisca.fr/) is free and is published under the [GNU Affero General Public Licence](https://www.gnu.org/licenses/agpl.html) version 3 or later.
 
-[OpenFisca](https://www.openfisca.fr/) is a versatile microsimulation software.
+## Project Components
 
-It is a free software published under the [GNU Affero General Public Licence](https://www.gnu.org/licenses/agpl.html) version 3 or later.
+OpenFisca is a modular project. Depending on your goals, you will install and interact with one or several of the OpenFisca Components.
 
-## What's the purpose?
+[![OpenFisca schema](https://cdn.rawgit.com/openfisca/openfisca-doc/eb7b9ce6f41d7ab2cb7bac08738cfb67d58c0c36/architecture.svg)](https://github.com/openfisca/openfisca-doc/blob/bird_eye_view/architecture.svg)
```

Comment:
Putting the hash here means updates to the file won't be automatically published. It also means that it will break if you rebase before merging.

This is not blocking as we can move this to target `master` the next time we need to make an update, but please at least be sure that the hash will still be valid before merging!